packages/plugins/RSS3/src/SNSAdaptor/index.tsx

```diff
@@ -16,7 +16,7 @@ const DonationsTabConfig: Plugin.SNSAdaptor.ProfileTab = {
     priority: 1,
     UI: {
         TabContent: ({ socialAddress, identity }) => {
-            if (!socialAddress?.address || !identity?.identifier?.userId || !identity.publicKey) return null
+            if (!socialAddress?.address || !identity?.identifier?.userId) return null
```

Comment:
Does this condition mean a social identity must be a NextID user? Please consult QA or PM.